docs/jsx.md

```diff
@@ -338,6 +340,41 @@ function SummaryView() {
 
 ---
 
+### A note on event listeners
+
+While JSX events are usually named using camelCase, lowercase names should be used when JSX is used with Mithril.
+
+```javascript
```

Comment:
the example is unnecessarily complex. I would suggest something like
```js
m("main", [
	m("input", { type: "text", oninput: ... }),
	m("input", { type: "button", value: "Click me", onclick: ... })
])
```

```jsx
<main>
	<input type="text" oninput={...}/>
	<input type="button" value="Click me" onclick={...}/>
</main>
```

Reply:
I agree, but I would after 3 months I would suggest this issue is closed.